Update README

PennyScissors opened the pull request.

aiyengar2 reviewed (commented):
Should we add something about the +up annotation too?

PennyScissors commented:
> Should we add something about the +up annotation too?

Updated to include information on the +up annotation and removed the `v` prefix from examples since it looks like we remove that when a package gets generated.

aiyengar2 reviewed (approved):
LGTM
